checkov/common/goget/github/get_git.py

```diff
@@ -71,7 +81,9 @@ def do_get(self) -> str:
     def _clone(self, git_url: str, clone_dir: str) -> None:
         self.logger.debug(f"cloning {self.url if '@' not in self.url else self.url.split('@')[1]} to {clone_dir}")
         with temp_environ(GIT_TERMINAL_PROMPT="0"):  # disables user prompts originating from GIT
-            if self.commit_id:
+            if self.branch:
```

Comment:
nice implementation :)
Just please add it this feature to the `--help` part

Reply:
added